### DISCUSSION TITLE
[小说]咖啡馆恐怖小说第三章，也是最后一章了

### DISCUSSION THREAD

**Discussion by aaa00073** [share121/inter-knot#685] [其他]
第三章
    今天晚上，小超市里进来了一位奇怪的顾客。

　　那人看模样是个学生，走进来的时候，两只手一直捏着脖子。

　　他身躯站的笔直，小心翼翼地走进来，说道：“老板，来一卷胶布，宽的，结实防水的那种。”

　　老板头也不抬地剪着指甲，“左边货架上，自己拿。”

　　“老板，你帮我拿一下，我有点不方便。”

　　老板抬头瞥了他一眼，随后站起身走去拿了一卷胶带。

　　“谢谢，能帮忙把胶带头撕开吗？”

　　老板在胶带上扣了半天，总算找到了，把胶带扯开，递给这个顾客。

　　“谢谢。”

　　对方拿起胶带，小心地腾出一只手，把胶带绕在脖子上。

　　一圈一圈，动作很慢，但是在脖子上缠了很多圈，甚至有的还从头顶缠过去。

　　一卷胶带都快用了一半了，他才停下来。

　　老板楞楞地看着陈往的行为，满脸的不理解。

　　“呼——”

　　陈往长舒一口气。

　　他微微转动脑袋。

　　嗯，还算稳妥，只要动作幅度不是太大，应该没什么问题。

　　付了钱，陈往对老板说道：“老板，能否借洗手间一用？”

　　“请便。”

　　已经超过10小时没上厕所了，陈往早就憋不住了。

　　上完厕所，在洗手台洗手的陈往看着镜子里自己的这幅模样。

　　不知为何，他感到很悲哀。

　　断了却没死，陈往不知道自己还能活多久。

　　他想洗个脸，但是怕水打湿了胶布。

　　水流声哗啦啦地，陈往却忽然浑身僵硬。

　　他一动不动，

　　直勾勾地盯着镜面。

　　镜子里没什么异常，

　　但陈往分明感受到，有人从身后抱住了自己！

　　视线往下瞥，陈往看到，自己胸口环绕着的两条手臂。

　　苍白而无血色。

　　手指纤细，陈往认得出来。

　　是那个女人！

　　她之前不是消失了，而是缠上了自己！

　　可镜子根本没有把她映照出来。

　　陈往内心在挣扎，

　　他要不要装作没看见？

　　然而就在这时，耳边忽然一阵阴凉。

　　“客人，讲一个故事吧，”

　　“讲给我听……”

　　陈往吞了口唾沫，还是用之前那种应付式的语气，说道：“要，什么样的故事？”

　　女人却没有回答他的问题，而是说了一句无关的话。

　　“从别人的故事里出来……回到自己的故事中……”

　　在这句话之后，拥抱的感觉慢慢消失了。

　　离开了吗？

　　但陈往还是等了好几分钟，才缓缓回头瞥了一眼。

　　的确消失了，但周围已经陷入了一片黑暗。

　　在漆黑中走了两步，陈往发现，这里什么都没有。

　　就像之前没逃出来那个地方一样。

　　陈往的心里正在疑惑。

　　就在此时，黑暗慢慢淡去了。

　　然后，陈往发现，自己正站在马路边，根本不在超市洗手间。

　　“这是，怎么回事？”

　　“我怎么到这里来了？”

　　陈往感受到一种不安。

　　长时间的紧张已经让他身心俱疲。

　　他现在只想回家。。

　　然而就在此刻，黑暗再次笼罩了周围。

　　一切都被遮蔽。

　　陈往已经懵了，一次又一次，到底是怎么回事？为什么自己一直被不断出现的黑暗笼罩？

　　他回想起女人刚才说的话。

　　“从别人的故事出来……”

　　陈往忽然想到了什么。

　　他向黑暗中某个方向跑去，他要去验证心中的猜想。

　　很快，一家店门出现在陈往面前。

　　深夜咖啡馆。

　　拥抱的感觉再次出现了。

　　陈往向咖啡馆靠近一步，绕在身上的双臂忽然勒紧，让他呼吸受困难。

　　似乎是在阻止他前进。

　　陈往表情痛苦，内心却是狂喜。

　　他似乎猜对了。

　　这个女人，跟这个咖啡馆，根本不是同一方的，而是对抗的！

　　虽然自己离开了咖啡馆，那女人仍然在称呼自己为“客人”！

　　这是在提醒他，自己根本没有摆脱咖啡馆。

　　而脱离的方法，女人也提醒他了，就是讲故事！

　　陈往心中的恐惧并没有消退，但决心正在增加。

　　他不敢肯定自己的猜想正确，但他从来自信于自己的直觉。

　　他敢赌一把！

　　既然要讲故事，那就讲真实的故事，亲历的故事，就像女人之前那样。

　　走进深夜咖啡馆，陈往拉来一张椅子坐下，面向黑暗。

　　他缓缓开口：

　　“我是个普普通通的高中生。”

　　“那天夜里，我想去尝试一杯手磨咖啡，于是走进了咖啡馆。”

　　“我碰见了咖啡馆的另一位客人，这里似乎只有她一个人。”

　　“我品尝了桌上的咖啡，并听另一位客人讲起了故事。”

　　“可是讲到最后，对方的脑袋竟然掉了下来！”

　　“我感到非常恐惧，一股脑逃跑了。”

　　“我逃了出来，回到了现实，我欣喜如狂。”

　　“突然间，我感觉脖颈发痒。”

　　“好像，我的头也快掉下来了。”

　　陈往说完了，紧紧盯着咖啡馆内小片能看见的区域。

　　见许久没有反应，陈往疑惑：“难道我猜错了？”

　　然而下一秒，视线开始扭曲，咖啡馆开始迅速倒退。

　　它变得模糊不清，

　　不断后退，

　　不断后退，

　　在视线中缩成了一个点。

　　消失了。

　　就在这一刻，陈往忽然感觉一切都开始明晰起来。

　　“我这是，成功了？”陈往感到不可置信。

　　黑暗依旧笼罩，但是黑暗中的一切都变得可以被陈往感知，甚至比眼睛看的还清楚。

　　此刻他才发现，自己仍然站在街道上。

　　几位路人走过他身边，但是身处于黑暗中的陈往，完全没有被察觉，甚至路人直接穿过了他。

　　陈往心念一动，黑暗以陈往为中心，完全收回，就像倒放的爆炸一样。

　　“啊，对不起对不起。”

　　由于陈往突然出现，一位路人不慎撞到了他，对方赶忙道歉。

　　走的时候，对方还在挠头。

　　陈往呼出一口气：“这一切，算是结束了吗？”

　　陈往回想过去十多个小时发生的一切。

　　他发现了一个有意思的地方。

　　“那个女人为什么讲了两个故事？回想起来，我出来的过程中，她缠上了我。”

　　“所以讲个故事，是为了让我们两个都离开？”

　　“继续往下推，如果我的脖子断开也是她所为的……”

　　“那岂不是说明，她是给我一个讲真实故事的机会？”

　　“她算是害了我，还是救了我？”

　　陈往越想越深，丝毫没察觉到一双苍白的手再次环住了他。

**Comment by kongbaodegou** [comment]
好看
